e2e_tests/tests/experiment/test_allocation_csv.py

```diff
@@ -84,7 +84,9 @@ def test_tensorboard_experiment_capture() -> None:
     assert task_id is not None
     clu.utils.wait_for_task_state("tensorboard", task_id, "TERMINATED")
 
-    end_time = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+    # Ensure that end_time captures tensorboard
+    end_time = (datetime.now(timezone.utc) + timedelta(minutes=1)).strftime("%Y-%m-%dT%H:%M:%SZ")
+    print(end_time)
```

Comment:
nit: Is this leftover from debugging? Good idea to print it, though I would prefer a more descriptive print statement than just `end_time`.